scripts/prohibit-class-name-hardcode-in-strings

```diff
@@ -7,6 +7,6 @@ source "$self_dir"/utils.sh
 
 class_names=$(git ls-files '*.java' | xargs basename -s .java | sort | paste -sd'|')
 
-if git grep -EHn  '^[^"]*"[^"$]*\<('"$class_names"')\>' -- '*.java' ':(exclude)./frontendDefs'; then
+if git grep -EHn  '^[^"]*"[^"$]*\<('"$class_names"')\>' -- '*{^Bundle}.java' ':(exclude)./frontendDefs'; then
```

Comment:
Is `*{^Bundle}.java` the same as `:(exclude)*Bundle.java`? :thinking:

Reply:
yup, but I couldn't combine it with `:(exclude)./frontendDefs` :confused: